### ISSUE TITLE
Controlled list for method types?

### ISSUE THREAD

**Issue by kobbejager** [E-RIHS/schema#11]
In the method schema there is a method_type field that is still an "examples" array. This probably needs to be converted into a controlled list. Moreover, if we expect to have methods for ARCHLAB services, more values will be required.

```json
"method_type": {
    "title": "Method Type",
    "description": "The type of method this represents",
    "type": "string",
    "examples": [
        "Analysis Method",
        "Technical Examination",
        "Research Approach"
    ]
},
```

**Comment by jpadfield** [comment]
We added this field in as logically it makes sense to use it. As with others yes a controlled list would be required for it to be useful, but I am not sure what the scope should be. I have a had a look at some existing options but have not found any easy answers. 

The issue here is also that logically the concept of a Method could also be extended to "Research Approach", which we have listed but may actually be a broader term.

Also, is there overlap between technical examination and analysis now given various modern techniques such as scanning XRF.

We are considering "Service Methods" here so we can limit things a bit ....
